lib/hello_web/controllers/fallback_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HelloWeb.FallbackController do
```

Comment:
@keliumJU what is the purpose of a fallback controller?

Reply:
Fallback controller what is intended is that developers avoid the repetition of code, in this case we control the error return that should not be explicit, by making use of the action_fallback it is enough to perform the error return.